sql/src/main/antlr/OpenDistroSQLIdentifierParser.g4

```diff
@@ -44,10 +44,18 @@ alias
 
 qualifiedName
     : ident (DOT ident)*
+    | keywordsCanBeId
```

Comment:
Can we also add functionNameBase here as before? https://github.com/opendistro-for-elasticsearch/sql/blob/master/legacy/src/main/antlr/OpenDistroSqlParser.g4#L237. Although this is a separate g4 file, I guess it may have access to rules in parent g4.

Reply:
It looks like it cannot read the `functionNameBase` so I added them separately. Do they have heritance relation, I only see `OpenDistroSQLParser` imports Identifier parser file

Reply:
I see. Thanks!